actioncable/test/subscription_adapter/postgresql_test.rb

```diff
@@ -22,6 +22,7 @@ def setup
       ActiveRecord::Base.connection
     rescue
       skip "Couldn't connect to PostgreSQL: #{database_config.inspect}"
+      @rx_adapter = @tx_adapter = nil
```

Comment:
Won't actually zap the warnings because `skip` aborts the test run with an exception raise, so this line will never be hit.

Committed a fix for this separately in https://github.com/rails/rails/commit/dacfdf03d34b8f9a3b81ef2219bf49307f90257e because I wanted to look into the deadlock and race conditions that happened periodically in our test rig, but got distracted by the warnings.